src/components/HistoryTables/MessageHistory/Detail/index.tsx

```diff
@@ -47,17 +48,17 @@ export default function MessageDetail(props: MessageDetailProps) {
   const [seeMore, setSeeMore] = useState(false)
   const { message, error, loading, pending } = useMessage(cid)
 
-  const { data: gasQuery } = useGasCostQuery({
+  const { data: gasQuery, error: gasStateError } = useGasCostQuery({
     variables: { cid },
     pollInterval: 10000
   })
-  const { data: msgRcptQuery } = useMessageReceiptQuery({
+  const { data: msgRcptQuery, error: msgRcptError } = useMessageReceiptQuery({
     variables: { cid },
     pollInterval: 10000
   })
 
   const transactionFee = useMemo<string>(() => {
-    if (pending || !gasQuery?.gascost?.totalCost) return 'Pending...'
+    if (pending) return 'Pending...'
     const cost = gasQuery?.gascost?.totalCost
     return cost
       ? `${makeFriendlyBalance(new FilecoinNumber(cost, 'attofil'))} FIL`
```

Comment:
If you return `'Pending...'` on `!gasQuery?.gascost?.totalCost`, the second part of the ternary statement (where it returns `''`) will never be hit. If we do want to return `'Pending...'` when cost is not yet known, we should do this:

```javascript
const cost = gasQuery?.gascost?.totalCost
return cost && !pending
  ? `${makeFriendlyBalance(new FilecoinNumber(cost, 'attofil'))} FIL`
  :  'Pending...'
```

Reply:
@Schwartz10 let me know which you prefer

Reply:
I prefer not _only_ relying on `pending` as a prop to determine if `gasQuery?.gascost?.totalCost` exists, since there is a state where the message is not pending, but the gascost has still yet to be computed. So i'd go with the version in the comment rather than the one commited

Reply:
It's not really a matter of relying on pending as a prop to determine if `gasQuery?.gascost?.totalCost` exists.

The question is more what we want to show in either case.

In the previous version, it basically said:
- Show "Pending..." if `gasQuery?.gascost?.totalCost` doesn't exist or we are pending
- And then show "" if `gasQuery?.gascost?.totalCost` doesn't exist

The latter would never be hit due to the previous statement.